runtime/ms-rest-azure/index.d.ts

```diff
@@ -26,6 +36,9 @@ export class AzureServiceClient extends msRest.ServiceClient {
     * Long Running Operations. Default value is 30.
   */
   constructor(credentials: msRest.ServiceClientCredentials, options: AzureServiceClientOptions)
+  getLongRunningOperationResult(resultOfInitialRequest: msRest.HttpOperationResponse<T>, options?: LongRunningUrlBasedRequestPrepareOptions | LongRunningPathTemplateBasedRequestPrepareOptions, optionalCallback?: msRest.ServiceCallback<TResult>): void | Promise<TResult>;
+  sendLongRunningRequest(options: LongRunningUrlBasedRequestPrepareOptions | LongRunningPathTemplateBasedRequestPrepareOptions, optionalCallack?: msRest.ServiceCallback<TResult>): void | Promise<TResult>;
+  sendLongRunningRequestWithHttpOperationResponse(options: LongRunningUrlBasedRequestPrepareOptions | LongRunningPathTemplateBasedRequestPrepareOptions): Promise<msRest.HttpOperationResponse<T>>;
```

Comment:
These 3 methods enables a user to manually send a request that is long running and the method will automatically poll.